src/FormHelperText/index.tsx

```diff
@@ -0,0 +1,69 @@
+/** @jsx jsx */
+/** @jsxFrag React.Fragment */
+import * as React from "react";
+import * as typography from "../typography";
+import { colors } from "../colors";
+import { css, jsx } from "@emotion/core";
+import { IconInfoSolid } from "../icons/IconInfoSolid";
```

Comment:
Can we use `IconInfo` here instead pls? 🙏 
```suggestion
import { IconInfo } from "../icons/IconInfo";
```

Reply:
Sure can @jglovier !